### DISCUSSION TITLE
Theme's colour for SVG - no change in default colours for text inside an SVG

### DISCUSSION THREAD

**Discussion by kuatroka** [saadeghi/daisyui#2837] [Q&A] [answered]
As in the title. 
I'm using a theme from the installation of **'shadcn-svelte'.** And I also use the **<Timeline>** component from DaisyUI. 
With html tags all is great, but <Timeline> has an svg in it and this what breaks for me when toggling the Dark/Light theme. I disabled DaisyUI themes in **tailwind.config.js** so everywhere else the theme is applied nicely a part from other component (a chart) that also uses SVG. See the gif 1. I tried to explicitly add a primary colour to the text part of an `html` with text, but it messes up the Light mode. See gif 2. 
I think what happens is the SVG doesn't change the colour for text that's inside the SVG and I don't know how to about it. See the gifs plus links to my code. Any help or direction would be great. Thanks

![svg-colour](https://github.com/saadeghi/daisyui/assets/9093546/4313c18f-b4e8-4b42-801e-c149d115f9a1)

**Assigning text colour explicitly** 
![svg-colour2](https://github.com/saadeghi/daisyui/assets/9093546/b1d00a2f-1f77-4056-903f-8ced5659d2e0)

[code](https://github.com/kuatroka/mi-ui-1/blob/main/src/routes/layerchart/%2Bpage.svelte)

**Comment by kuatroka** [comment]
Any ideas how can I use primary, secondary, etc colours from shadcn-svelte in Daisy's svgs?

**Comment by kuatroka** [comment, marked as answer]
solution is to add `bg-background`

**Comment by saadeghi** [comment]
> solution is to add `bg-background`

That's class name from shadcn, not daisyUI or Tailwind.

**Reply by kuatroka** [reply]
Indeed, but I don't know how else to solve it. It's the name that is defined in the app.pcss file. And by default the when using the Daisy's timeline, al every colour but the box's background changes when toggling the theme. Meaning, everything works, only the box background doesn't change
